src/test/java/hudson/plugins/parameterizedtrigger/test/FileBuildParameterFactoryTest.java

```diff
@@ -245,22 +257,31 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
             }
         });
 
-        // add Trigger builder, with file parameter factory
-        projectA.getBuildersList().add(createTriggerBuilder(projectB, NoFilesFoundEnum.SKIP, "UTF-8"));
+        // SECURITY-170: need to allow multibyte params that can't be traditionally declared.
```

Comment:
> I had coped things over in vi

Srsly? :-) Well I know at least NetBeans correctly loads & saves all project sources in the encoding declared in the POM…would hope IDEA does the same.